ibridges/cli/other.py

```diff
@@ -220,4 +221,30 @@ def run_command(cls, args):
             with open(args.output, "w", encoding="utf-8") as handle:
                 handle.write(json_str)
 
-CLI_BULTIN_COMMANDS=[CliShell, CliAlias, CliInit, CliSetup]
+class CliGui(BaseCliCommand):
+    """Subcommand to open the iBridges GUI."""
+
+    names = ["gui"]
+    description = "Start the iBridges GUI."
+    examples = ["gui"]
+
+    @staticmethod
+    def run_shell(session, parser, args):
+        """Run init is not available for shell."""
+        raise NotImplementedError()
+
+    @classmethod
+    def run_command(cls, args):
+        """Start the GUI."""
+        parser = cls.get_parser(argparse.ArgumentParser)
+
+        if (importlib.util.find_spec("ibridgesgui")) is not None:
+            from ibridgesgui.__main__ import main  # type: ignore # pylint: disable=E0401, C0415
+            main()
+        else:
+            parser.error("'ibridgesgui' is not installed. Please install with:")
```

Comment:
This should be one line, since the parser.error will exit on its own, you can remove the `sys.exit`.